src/db.js

```diff
@@ -75,12 +76,13 @@ export class DB {
    * !> Any returned Rows object needs to be fully
    * iterated over or discarded by calling
    * `.done()`.
+   *
```

Comment:
Can you remove the extra newline in the comment? (This looks nicer, especially in the generated docs 😅)

Reply:
Fixed.